jenkins_jobs/build_host_os/script.sh

```diff
@@ -29,8 +24,6 @@ fi
 # Fetch pull requests in case this job was triggered by one
 git fetch origin +refs/pull/*:refs/remotes/origin/pr/*
```

Comment:
No, that's not it; I think GitHub messed with the commit order again. Nevermind then.